plugins/scaffolder/api-report.md

```diff
@@ -230,6 +230,11 @@ export type RouterProps = {
     editor?: boolean;
     actions?: boolean;
   };
+  customFilters?: Array<{
```

Comment:
This API feels a little bit awkward tbh. It's not entirely clear to me what I get by providing these values if I just read this in isolation. I'm thinking it's better to revisit this later though, and focus on the other concerns first

Reply:
I have changed this to take an array of custom filter component, instead of taking in the properties needed for the EntityGenericPicker. This way, we can reuse any filter component created for the catalog page.